lib/core/util.js

```diff
@@ -562,6 +562,26 @@ function errorRequest (client, request, err) {
 const kEnumerableProperty = Object.create(null)
 kEnumerableProperty.enumerable = true
 
+const normalizeMethodRecord = {
```

Comment:
I dont like the name as the first part is a verb, indicating a function
Rename to normaledMethodRecords?